packages/components/src/components/vertical-tab/vertical-tab.jsx

```diff
@@ -1,14 +1,17 @@
-import classNames                      from 'classnames';
-import PropTypes                       from 'prop-types';
-import React                           from 'react';
-import { connect }                     from 'Stores/connect';
-import { VerticalTabContentContainer } from './vertical-tab-content-container.jsx';
-import { VerticalTabHeaders }          from './vertical-tab-headers.jsx';
+import classNames                  from 'classnames';
+import PropTypes                   from 'prop-types';
+import React                       from 'react';
+import VerticalTabContentContainer from 'Components/vertical-tab/vertical-tab-content-container.jsx';
+import VerticalTabHeader           from 'Components/vertical-tab/vertical-tab-header.jsx';
+import VerticalTabHeaders          from 'Components/vertical-tab/vertical-tab-headers.jsx';
+import VerticalTabHeaderTitle      from 'Components/vertical-tab/vertical-tab-header-title.jsx';
+import VerticalTabLayout           from 'Components/vertical-tab/vertical-tab-layout.jsx';
+import VerticalTabWrapper          from 'Components/vertical-tab/vertical-tab-wrapper.jsx';
 
 class VerticalTab extends React.Component {
     constructor(props) {
         super(props);
-        this.setSelectedIndex(props);
+        this.state = { modal_index: props.modal_index || 0 };
```

Comment:
How come VerticalTab has a `modal_index` ? Is it because VerticalTab can only be used inside a modal?

Reply:
The name is confusing as it actually refers to the selected vertical tab rather than the index of a modal. Let's rename it?

Reply:
@aaron-binary Sure 👍